boards/arm/raspberrypi_pico/raspberrypi_pico-pinctrl.dtsi

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2021, Yonatan Schachter
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/rp2040-pinctrl.h>
+
+&pinctrl {
+};
```

Comment:
please, add board after all drivers (drivers and board content should not be mixed in the same commit)